shells/web-shell/index.html

```diff
@@ -17,6 +17,7 @@
   <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
 
   <script src="../configuration/csp.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/@magenta/music@1.4.2"></script>
```

Comment:
We should put this in the component, I believe, rather than the web shell directly.

Reply:
One example way to do this is here: https://github.com/PolymerLabs/arcs/blob/master/modalities/dom/components/elements/good-map.js#L19

Reply:
Great example. We were trying to get something quick-and-dirty.